src/common/Loader/styled.js

```diff
@@ -0,0 +1,87 @@
+import styled from "styled-components";
+import { ReactComponent as Loading } from "./icon-spinner.svg"
+
+export const Wrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: stretch;
+    align-items: center;
+    max-width: 1368px;
+    margin: 0 auto;
+
+    @media ( max-width: ${({ theme }) => theme.breakpoints.small}px) {
+        margin: 24px 16px;
+    };
+`;
+
+export const Header = styled.h1`
+    align-self: flex-start;
+    margin: 56px 56px 120px;
+    font-size: 36px;
+    font-style: normal;
+    font-weight: 600;
```

Comment:
I think you can skip these 2 lines and it still going to work ok.

Reply:
i think that default font weight would not be  600 so i wrote it:)

Reply:
We can actually skip most of the code.
Instead of almost 100 lines, a few are enough. 

`import styled from "styled-components";
import { ReactComponent as Loading } from "./icon-spinner.svg"

export const Wrapper = styled.div`
    max-width: 1368px;
    margin: 0 auto;
`;

export const Header = styled.h1`
    margin: 56px 0 120px 20px;
    font-size: 36px;
`;

export const LoadingIcon = styled(Loading)`
    animation-name: rotate;
    animation-duration: 1s;
    animation-iteration-count: infinite;
    animation-timing-function: linear;
    margin: auto auto; 
    display: block;
`;`

Reply:
Or even simpler: 

`import styled, {keyframes} from "styled-components";
import { ReactComponent as Loading } from "./icon-spinner.svg"

const rotate = keyframes`
    to {
        transform: rotate(360deg);
    }
`;
export const Wrapper = styled.div`
    max-width: 1368px;
    margin: 0 auto;
`;

export const Header = styled.h1`
    margin: 56px 0 120px 20px;
    font-size: 36px;
`;

export const LoadingIcon = styled(Loading)`
    margin: auto auto; 
    display: block;
    animation: ${rotate} 1s linear infinite;
`;`